controllers/chat/group/leaveGroupMember.js

```diff
@@ -34,20 +34,25 @@ const leaveGroupMember = async (req, res) => {
       const textDefaultUser = `${ownUser.username} left this group`;
       const members = betterChannelMember.map((member) => member.user_id);
 
-      await currentChannel.sendMessage({
-        text: textDefaultUser,
-        own_text: textOwnUser,
-        other_text: textTargetUser,
-        better_type: 'leave_group',
-        type: 'system',
-        user_id: userId,
-        only_to_user_show: userId,
-        disable_to_user: false,
-        is_from_prepopulated: true,
-        system_user: userId,
-        isSystem: true,
-        members: members
-      });
+      await currentChannel.sendMessage(
+        {
+          text: textDefaultUser,
+          own_text: textOwnUser,
+          other_text: textTargetUser,
+          better_type: 'leave_group',
+          type: 'system',
+          user_id: userId,
+          only_to_user_show: userId,
+          disable_to_user: false,
+          is_from_prepopulated: true,
+          system_user: userId,
+          isSystem: true,
+          members: members
+        },
+        {
+          skip_push: true
+        }
+      );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change seems to be logically correct as it adds a `skip_push: true` option to the `sendMessage` function call. This will prevent push notifications from being sent when a user leaves a group. However, please ensure that this behavior is intended and acceptable in all use cases where the `leaveGroupMember` function is called. If there are scenarios where a push notification should be sent when a user leaves a group, you might need to make `skip_push` configurable rather than hardcoding it to `true`.

```diff
-         {
-           skip_push: true
-         }
+         {
+           skip_push: shouldSkipPush || true
+         }
```
In the above diff, `shouldSkipPush` would be a new parameter for the `leaveGroupMember` function that allows callers to specify whether push notifications should be skipped.


<!-- This is an auto-generated comment by OSS CodeRabbit -->